x-pack/packages/kbn-ecs-data-quality-dashboard/impl/data_quality/data_quality_panel/pattern/helpers.ts

```diff
@@ -160,47 +165,14 @@ export const getSummaryTableItems = ({
     ilmPhase: ilmExplain != null ? getIlmPhase(ilmExplain[indexName]) : undefined,
     pattern,
     patternDocsCount,
+    sizeInBytes: getSizeInBytes({ stats, indexName }),
   }));
 
-  return sortBy('docsCount', summaryTableItems).reverse();
-};
-
-export const getDefaultIndexIncompatibleCounts = (
-  indexNames: string[]
-): Record<string, number | undefined> =>
-  indexNames.reduce<Record<string, number | undefined>>(
-    (acc, indexName) => ({
-      ...acc,
-      [indexName]: undefined,
-    }),
-    {}
-  );
-
-export const createPatternIncompatibleEntries = ({
-  indexNames,
-  patternIncompatible,
-}: {
-  indexNames: string[];
-  patternIncompatible: Record<string, number | undefined>;
-}): Record<string, number | undefined> =>
-  indexNames.reduce<Record<string, number | undefined>>(
-    (acc, indexName) =>
-      indexName in patternIncompatible
-        ? { ...acc, [indexName]: patternIncompatible[indexName] }
-        : { ...acc, [indexName]: undefined },
-    {}
-  );
-
-export const getIncompatible = (
-  patternIncompatible: Record<string, number | undefined>
-): number | undefined => {
-  const allIndexes = Object.values(patternIncompatible);
-  const allIndexesHaveValues = allIndexes.every((incompatible) => Number.isInteger(incompatible));
-
-  // only return a number when all indexes have an `incompatible` count:
-  return allIndexesHaveValues
-    ? allIndexes.reduce<number>((acc, incompatible) => acc + Number(incompatible), 0)
-    : undefined;
+  if (sortByDirection === 'desc') {
+    return sortBy(sortByColumn, summaryTableItems).reverse();
```

Comment:
I'll stop commenting on these after this.. re the `orderBy` vs `sort().reverse()`. And once again, not sure which would be more performant